<launch>
  <!-- HgROS -->  
  <node name="HgROS" pkg="hg_cpp" type="hg_ros" output="screen">
	<rosparam file="$(find hg_cpp)/params/test_servo.yaml" command="load" />			
	<param name="simulate" value="false"/>
  </node>
</launch>
